tests/ui_tests/test_bundle_list_page.py

```diff
@@ -107,5 +165,93 @@ def test_two_bundles(bundle_archives: List[str], page: BundleListPage):
         page.upload_bundle(bundle_archives[0])
     with page.table.wait_rows_change():
         page.upload_bundle(bundle_archives[1])
-    rows = page.table.row_count
-    assert rows == 2, f'Row amount should be 2, but only {rows} is presented'
+    with allure.step('Check amount of rows'):
+        rows = page.table.row_count
+        assert rows == 2, f'Row amount should be 2, but only {rows} is presented'
+
+
+def test_open_bundle_from_table(page: BundleListPage, upload_bundle: Bundle):
+    """Test open bundle object page from list of bundles"""
+    with allure.step('Open bundle object page from bundle list'):
+        page.click_bundle_in_row()
+    with allure.step('Check object page is opened'):
+        object_page = BundlePage(page.driver, page.base_url, upload_bundle.id)
+        object_page.wait_page_is_opened()
+
+
+def test_open_main_menu_on_bundle_page(page: BundleListPage, upload_bundle: Bundle):
+    with allure.step('Open bundle object page'):
+        object_page = BundlePage(page.driver, page.base_url, upload_bundle.id)
+        object_page.open()
+    object_page.open_main_menu()
+    object_page.check_all_fields_presented()
+
+
+@pytest.mark.full()
+@pytest.mark.usefixtures('upload_bundle')
+def test_open_adcm_main_menu(page: BundleListPage):
+    """Open main menu by clicking on the menu icon in toolbar"""
+    page.find_and_click(BundleListLocators.Tooltip.apps_btn)
+    AdminIntroPage(page.driver, page.base_url).wait_page_is_opened()
+
+
+@pytest.mark.full()
+@pytest.mark.usefixtures("_create_connected_objects")
+def test_delete_provider_with_created_cluster(page: BundleListPage):
```

Comment:
Do you mean `test_delete_bundle_with_created_cluster` ?

Reply:
Yeah, thanks, fixed